region/crateria/east/The Moat.json

```
@@ -690,7 +647,8 @@
             "HiJump"
           ]}
         ]}
-      ]
+      ],
+      "collectsItems": [3]
```

All of these can avoid the item. The gravity jump looks precise, but it is probably still easier than whatever tech would exist to purposefully skip items
```suggestion
      ]
```

---

In that case I think it makes sense to change the strat to go 2->1 instead of 2->3, since I'm not sure it would make sense to end up at node 3 without having collected the item?